src/spec-configuration/containerFeaturesOCI.ts

```diff
@@ -60,34 +60,34 @@ export function getOCIFeatureSet(output: Log, identifier: string, options: boole
     return featureSet;
 }
 
-export function getFeatureRef(output: Log, identifier: string): OCIFeatureRef {
+export function getFeatureRef(output: Log, resourceAndVersion: string): OCIFeatureRef {
 
     // ex: ghcr.io/codspace/features/ruby:1
```

Comment:
Suggestion: Change example to `// ex: ghcr.io/devcontainers/features/ruby:1` ?

Reply:
Agreed. Once we open source the repo i'm hoping to s/codspace/devcontainers/g